raytracing/imagingpath.py

```diff
@@ -299,6 +300,17 @@ def lagrangeInvariant(self, ray1=None, ray2=None, z=0):
 
         return super(ImagingPath, self).lagrangeInvariant(z=z, ray1=ray1, ray2=ray2)
 
+    @property
+    def displayRange(self):
```

Comment:
We need a docstring even if the format is not finalized yet explaining the rationale for these choices.